azure-iot-provisioning-devicesdk/samples/register_symmetric_key.py

```diff
@@ -0,0 +1,52 @@
+import os
+import logging
+from azure.iot.provisioning.devicessdk.security.sk_security_client import SymmetricKeySecurityClient
+from azure.iot.provisioning.devicessdk.device.registration_client_factory import (
+    create_from_security_provider,
+)
```

Comment:
importing a single function is generally a bad look. Ideally a module is imported, and you use module.function_name(). See how the latest Hub Device SDK samples use auth for an idea of what I'm talking about. But even making that change here won't be enough, because the module "registration_client_factory" is too unwieldly.

I have some ideas for adjustments to API shape, but don't worry about it now - once the featureset is more fleshed out, we can revisit the shape of the imports, and the exact instantiation process.